manifests/virtualenv.pp

```diff
@@ -86,24 +86,27 @@
       }
     }
 
-    # Virtualenv versions prior to 1.7 do not support the
-    # --system-site-packages flag, default off for prior versions
-    # Prior to version 1.7 the default was equal to --system-site-packages
-    # and the flag --no-site-packages had to be passed to do the opposite
+    # Virtualenv handling of site packages has changed over time, both its
+    # default and the flags to control it. See inline comments for details.
     $_virtualenv_version = getvar('virtualenv_version') ? {
       /.*/ => getvar('virtualenv_version'),
       default => '',
     }
-    if (( versioncmp($_virtualenv_version,'1.7') > 0 ) and ( $systempkgs == true )) {
-      $system_pkgs_flag = '--system-site-packages'
-    } elsif (( versioncmp($_virtualenv_version,'1.7') < 0 ) and ( $systempkgs == false )) {
-      $system_pkgs_flag = '--no-site-packages'
-    } else {
-      $system_pkgs_flag = $systempkgs ? {
-        true    => '--system-site-packages',
-        false   => '--no-site-packages',
-        default => fail('Invalid value for systempkgs. Boolean value is expected')
-      }
+    $system_pkgs_flag = $systempkgs ? {
+      # --system-site-packages was introducd in 1.7, and --no-site-packages
+      # became the default
+      true    => versioncmp($_virtualenv_version,'1.7') > 0 ? {
```

Comment:
Note: I thiiink that this should probably be `>=`, however I believe the current spelling has the same error.